.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+name: ansible-lint
+"on":
+  pull_request:
+    branches: ["main", "stable-*"]
+jobs:
+  build:
+    name: Ansible Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run ansible-lint
```

Comment:
A checkout step needs to be added before here. There was a breaking change in the action that started requiring that.